### PR TITLE
Add: added appium logo for webdriveragent (https://github.com/appium/…

### PR DESCRIPTION
**Pull request by harsha509**
Added appium logo for WebDriverAgent. Attached Emulator screenshots (iPad Pro, iPhoneXr)
Snip20190613_2
Snip20190613_1

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
Thanks. We'd also expect to see this icon present for WebDriverAgent-Runner app as well.

File: build/index.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
+"use strict";
```
Comment (mykola-mokhnach) [inline build/index.js]: the content of the `build` folder should be ignored

**Comment by mykola-mokhnach** [comment]
Is it expected that xcassets are duplicated in the root folder and in the resources folder?

File: build/index.js (hunk 1 above)
Comment (harsha509) [reply build/index.js]: Sure, I'll exclude the build folder

**Comment by harsha509** [comment]
> Is it expected that xcassets are duplicated in the root folder and in the resources folder?

NO.I was testing the feature in all emulators. I will remove the duplicates from xcode project.

**Comment by dpgraham** [comment]
@mykola-mokhnach does this look good to you now?

**Comment by harsha509** [comment]
Hi @dpgraham , @mykola-mokhnach 

Am able to integrate icon in xcode project and am able to see the icon for IntegrationApp when launched in simulator. 

I linked my changes in WebDriverAgent with my github URL in appium-xcuitest-driver-> package.json and running 1) npm run clean 2) npm run e2e-test:web
but it stills shows the same old app icon. But when i run in xcode app icon is set to IntegrationApp successfully

If am doing wrong, please suggest me and provide me some time to achieve it.

Thanks.

**Comment by mykola-mokhnach** [comment]
The problem is that the Runner app is not really an app in its classic representation. It's a testing bundle. That is why it is not very clear how to assign an icon to it (and whether it is possible in general).

**Comment by harsha509** [comment]
> The problem is that the Runner app is not really an app in its classic representation. It's a testing bundle. That is why it is not very clear how to assign an icon to it (and whether it is possible in general).

Thanks for the info @mykola-mokhnach , i will get back to you with required info on the possibility on achieving this. So that we can close or move ticket to any other status.

Thanks again :)

**Comment by krishtoautomate** [comment]
how come companies like sauce labs are able to add icon?

**Pull request closed without merging**